manifests/b/BenHollis/PNGGauntlet/3.1.2/BenHollis.PNGGauntlet.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BenHollis.PNGGauntlet
+PackageVersion: 3.1.2

```

```suggestion
PackageVersion: 3.1.1
```
This must match Control Panel to prevent upgrade loops due to incorrect version in Control Panel.

---

You will also need to update the version folder from `3.1.2` to `3.1.1`